src/docs/arch-eviction.dox

```diff
@@ -64,23 +64,25 @@ a database via ::wiredtiger_open, or changed after open with
 WT_CONNECTION::reconfigure.
 
 The \c eviction_target configuration value is the level at
-which WiredTiger attempts to keep the overall cache usage.  Eviction worker
+which WiredTiger attempts to keep the overall cache usage. Eviction worker
```

Comment:
Not related to this line but just a place to put this comment: I think you want to update the generated documentation about the config values themselves in `dist/api_data.py`. That is the "real" documentation that describes these kinds of config interactions.